libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/EHorizonResultType.kt

```diff
@@ -0,0 +1,16 @@
+package com.mapbox.navigation.base.trip.model
+
+/**
+ * EHorizonResultType
+ */
+sealed class EHorizonResultType {
```

Comment:
Same as above.

Reply:
@mskurydin @SiarheiFedartsou could you help here? Are we envisioning extending this / adding more data as part of `EHorizonResultType` in the future?

Reply:
I would expect interface will evolve.